backend/maelstro/scripts/code_check.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"/..
+
+poetry run black --check . && \
```

Comment:
We could consider using pre-commit: https://pre-commit.com/

IMHO those files don't need to be inside the Python package.

Reply:
in this case, pre-commit would be too cumbersome

And for the scripts I really wonder. They are placed inside the package since they use the package dependencies. This is a hard choice.

Reply:
I understand you need it in the docker image make sense, but in the Python package seems weird for me.

Reply:
I understand you need it in Docker images, but having them in Python package seems weird to me. BTW not sure it is really important.